sktime_dl/deeplearning/tests/test_regressors.py

```diff
@@ -59,9 +60,22 @@ def test_regressor_forecasting(
     forecaster.fit(y_train)
     y_pred = forecaster.predict(fh)
 
-    # Compare the prediction to the test data
-    mse = np.sqrt(mean_squared_error(y_test, y_pred))
-    print("Error:", mse)
+    try:
```

Comment:
at some point we may also want to write a config file for the tests, so that we collect all settings/exceptions in a single place, I've started doing some here for sktime: https://github.com/alan-turing-institute/sktime/blob/master/sktime/tests/_config.py

Reply:
That looks solid, a revamp of the tests (for speed if nothing else) makes a lot of sense as a 0.3.0 target